### PR TITLE
fix(ImageProperty): Return true/false on set methods

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

File: Sources/Rendering/Core/ImageProperty/index.js [hunk 1]
```diff
@@ -86,14 +90,16 @@ function vtkImageProperty(publicAPI, model) {
   publicAPI.setComponentWeight = (index, value) => {
     if (index < 0 || index >= VTK_MAX_VRCOMP) {
       vtkErrorMacro('Invalid index');
-      return;
+      return false;
```
Comment (jourdain) [inline Sources/Rendering/Core/ImageProperty/index.js:93]: that was the issue FYI @laurennlam

File: Sources/Rendering/Core/VolumeProperty/index.js [hunk 2]
```diff
@@ -159,14 +173,16 @@ function vtkVolumeProperty(publicAPI, model) {
   publicAPI.setComponentWeight = (index, value) => {
     if (index < 0 || index >= VTK_MAX_VRCOMP) {
       vtkErrorMacro('Invalid index');
-      return;
+      return false;
```
Comment (jourdain) [inline Sources/Rendering/Core/VolumeProperty/index.js:176]: that was the issue FYI @llangholz89

File: Sources/Rendering/Core/ImageProperty/index.js (hunk 1 above)
Comment (llangholz89) [reply Sources/Rendering/Core/ImageProperty/index.js:93]: Aha! Mystery solved, thanks!

**Pull request merged**